service/k8sclient/kubectl/kubectl.go

```diff
@@ -56,31 +57,38 @@ func NewKubeCtl(ctx context.Context, kubeconfig string) (*KubeCtl, error) {
 	l := logger.Get(ctx)
 	l = l.WithField("component", "kubectl")
 
-	// Handle kubectl versions
-	cmd, err := getKubectlCmd(ctx)
-	if err != nil {
-		return nil, err
-	}
-
-	l.Infof("Using %q", strings.Join(cmd, " "))
-
 	if kubeconfig == "" {
+		// Handle kubectl versions
+		cmd, err := getKubectlCmd(ctx, "")
+		if err != nil {
+			return nil, err
+		}
 		return &KubeCtl{
 			l:   l,
 			cmd: cmd,
 		}, nil
 	}
 
+	var tmpDir, kubeconfigPath string
+	var err error
+
 	// Handle kubeconfig.
-	tmpDir, kubeconfigPath, err := saveKubeconfig(kubeconfig)
+	tmpDir, kubeconfigPath, err = saveKubeconfig(kubeconfig)
```

Comment:
Why did you replace `:=` with explicit variable declarations?

Reply:
oops, I was moving lines up and down and then forgot to remove :)